frontend/src/components/Profile/Profile.js

```diff
@@ -51,23 +69,26 @@ export default function Profile(props) {
                 <div className="follow-btn">
                     {
                         !(props.currentUser === props.username) &&
-                        <FollowingButton {...props} class='following-user'/>
+                        <FollowingButton username={props.username} refreshProfile={handleFollowChange} class='following-user' />
                     }
                     {errorMsg && <div className="error">{errorMsg}</div>}
                 </div>
-                <div className="edit-btn">
-                    <Link to={"/accounts/edit"}>
-                        <div className="gear-icon">
-                            <svg className="bi bi-gear" width="1.2em" height="1.2em" viewBox="0 0 16 16" fill="currentColor">
-                                <path fillRule="evenodd" d="M8.837 1.626c-.246-.835-1.428-.835-1.674 0l-.094.319A1.873 1.873 0 014.377 3.06l-.292-.16c-.764-.415-1.6.42-1.184 1.185l.159.292a1.873 1.873 0 01-1.115 2.692l-.319.094c-.835.246-.835 1.428 0 1.674l.319.094a1.873 1.873 0 011.115 2.693l-.16.291c-.415.764.42 1.6 1.185 1.184l.292-.159a1.873 1.873 0 012.692 1.116l.094.318c.246.835 1.428.835 1.674 0l.094-.319a1.873 1.873 0 012.693-1.115l.291.16c.764.415 1.6-.42 1.184-1.185l-.159-.291a1.873 1.873 0 011.116-2.693l.318-.094c.835-.246.835-1.428 0-1.674l-.319-.094a1.873 1.873 0 01-1.115-2.692l.16-.292c.415-.764-.42-1.6-1.185-1.184l-.291.159A1.873 1.873 0 018.93 1.945l-.094-.319zm-2.633-.283c.527-1.79 3.065-1.79 3.592 0l.094.319a.873.873 0 001.255.52l.292-.16c1.64-.892 3.434.901 2.54 2.541l-.159.292a.873.873 0 00.52 1.255l.319.094c1.79.527 1.79 3.065 0 3.592l-.319.094a.873.873 0 00-.52 1.255l.16.292c.893 1.64-.902 3.434-2.541 2.54l-.292-.159a.873.873 0 00-1.255.52l-.094.319c-.527 1.79-3.065 1.79-3.592 0l-.094-.319a.873.873 0 00-1.255-.52l-.292.16c-1.64.893-3.433-.902-2.54-2.541l.159-.292a.873.873 0 00-.52-1.255l-.319-.094c-1.79-.527-1.79-3.065 0-3.592l.319-.094a.873.873 0 00.52-1.255l-.16-.292c-.892-1.64.902-3.433 2.541-2.54l.292.159a.873.873 0 001.255-.52l.094-.319z" clipRule="evenodd"/>
-                                <path fillRule="evenodd" d="M8 5.754a2.246 2.246 0 100 4.492 2.246 2.246 0 000-4.492zM4.754 8a3.246 3.246 0 116.492 0 3.246 3.246 0 01-6.492 0z" clipRule="evenodd"/>
-                            </svg>
-                        </div>
-                        <button className="edit-button-component">
-                            Edit Profile
+                {
+                    (currentClient === username) &&
```

Comment:
Indentation: my eyes might be playing tricks on me but I think there's an extra space at the beginning of this line that doesn't make it quite aligned with the following line

Reply:
It's aligned, no extra space from my side